### PR TITLE
Fix autoload in composer.json

### PR DESCRIPTION
**Pull request by WinterSilence**
Move `yii\cs\`  into `autoload-dev` and add  `tests/` and `build/` namespaces

| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -99,8 +99,14 @@
...
-            "yii\\": "framework/",
-            "yii\\cs\\": "cs/src/"
+            "yii\\": "framework/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "yii\\cs\\": "cs/src/",
+            "yii\\build\\": "build/",
+            "yiiunit\\": "tests/"
```
Comment (samdark) [inline composer.json:109]: Why do we need PSR-4 autoload for these? It worked well before.

Comment (WinterSilence) [reply composer.json:109]: @samdark why do we need a third-party autoloader if Yii  have own autoloader? :) Mainly to shut up IDE's

**Review by schmunk42 — APPROVED** [review]
LGTM

**Comment by samdark** [comment]
Merged. Thanks.

**Pull request merged**